scripts/hooks/pre-commit

```diff
@@ -4,7 +4,13 @@
 
 STATUS=0
 
-./run_formatter.sh
+if [ -f ./run_formatter.sh ]; then
```

Comment:
We shouldn't need so many backups; when this commit goes through, there will never be a ./RunFormatter.sh